app/views/archive_faqs/_admin_index.html.erb

```diff
@@ -5,7 +5,7 @@
 
 <!--subnav-->
 <%= render "archive_faqs/filters" %>
-<%= render "admin/admin_nav" %>
+<%= render :partial => 'admin/admin_nav' %>
```

Comment:
Is there a reason `partial` is necessary here now? `<%= render "admin/admin_nav" %>` _should_ still work fine.